src/sprite.c

```diff
@@ -226,7 +226,24 @@ void spr_init_component_anim_state(SpriteComponent* comp, s16*** anim) {
     comp->scale.z = 1.0f;
 }
 
+
+// matches, but has a very questionable cast, spr_init_component_anim_state args are probably wrong
```

Comment:
Does this func actually match? If so it might be wise to remove the asm/NON_EQUIVALENT wrapper and just have this as a `// TODO improve match`

Reply:
yea it does match, no clue why I added the NON_EQUIVALENT thing here tbh lol